cmd/age/age.1

```diff
@@ -0,0 +1,162 @@
+.Dd June 28, 2020
+.Dt AGE 1
+.Sh NAME
+.Nm age
+.Nd a simple, modern and secure file encryption tool
+.Sh SYNOPSIS
+.Nm
+.Fl r Ar recipient
+.Op Fl a
+.Op Fl o Ar output
+.Op Ar input
+.Nm
+.Fl -decrypt
+.Op Fl i Ar key
+.Op Fl o Ar output
+.Op Ar input
+.Sh DESCRIPTION
+In the first synopsis form,
+.Nm
+encrypts
+.Ar input
+to the specified
+.Ar recipient .
+In the second synopsis form,
+.Nm
+decrypts
+.Ar input .
+.Pp
+The following options are available:
+.Bl -tag -width Ds -compact
+.It Fl o , Fl -output Ar output
+Write the result to the file at path
+.Ar output .
+.It Fl a , Fl -armor
+Encrypt to a PEM encoded format.
+.It Fl p , Fl -passphrase
+Encrypt with a passphrase.
+.It Fl r , Fl -recipient Ar recipient
+Encrypt to the specified
+.Ar recipient .
+Can be repeated.
+.It Fl d , Fl -decrypt
+Decrypt the input to the output.
+.It Fl i , Fl -identity Ar key
+Use the private key file at path
+.Ar key .
+Can be repeated.
+.El
+.Pp
+The
+.Ar input
+argument is optional and defaults to standard input. Only a single file may be
```

Comment:
Missed newline after period. This is necessary for sentence spacing, though I believe you're likely already aware of this and just missed it.